.meta/sinks/datadog_logs.toml.erb

```diff
@@ -40,15 +40,9 @@ description = "Datadog [API key](https://docs.datadoghq.com/api/?lang=bash#authe
 [sinks.datadog_logs.options.endpoint]
 type = "string"
 required = false
-default = "intake.logs.datadoghq.com:10516"
-examples = ["127.0.0.1:8080", "example.com:12345"]
+default = "https://http-intake.logs.datadoghq.com"
+examples = ["http://127.0.0.1", "https://example.com"]
 description = "The endpoint to stream logs to."
 
-<%= render(
-  "_partials/fields/_tls_connector_options.toml",
-  namespace: "sinks.datadog_logs.options",
-  can_enable: true,
-  enabled_default: true,
-  can_verify_certificate: true,
-  can_verify_hostname: true
-) %>
```

Comment:
I think it should be pretty quick to sort out these doc changes, so it's probably worth spending the 10 minutes to do them rather than risk this pull sitting around for longer than necessary..